Packages/Status/Sources/Status/Row/StatusRowView.swift

```diff
@@ -114,6 +114,16 @@ public struct StatusRowView: View {
         Text(status.spoilerText)
           .font(.body)
       }
+      if viewModel.displaySpoiler == false && !viewModel.status.spoilerText.isEmpty {
+        Button {
+          withAnimation {
+            viewModel.displaySpoiler = true
+          }
+        } label: {
+          Text("Show less")
```

Comment:
I think you can use the same button for the show less / more, with something like 
```
  Button {
          withAnimation {
            viewModel.displaySpoiler.toggle()
          }
        } label: {
          Text(viewModel.displaySpoiler ? "Show less" : "Show more")
        }
```

And then only the condition above `!viewModel.status.spoilerText.isEmpty` can remains.

Reply:
That looks a lot cleaner! So something like this?

```swift
if !viewModel.status.spoilerText.isEmpty {
  Text(status.spoilerText)
    .font(.body)
  Button {
    withAnimation {
      viewModel.displaySpoiler.toggle()
    }
  } label: {
      Text(viewModel.displaySpoiler ? "Show more" : "Show less")
  }
    .buttonStyle(.bordered)
}
if !viewModel.displaySpoiler { // Remove the "if/else" statement and make it just an if
  Text(status.content.asSafeAttributedString)
    .font(.body)
    .environment(\.openURL, OpenURLAction { url in
        routeurPath.handleStatus(status: status, url: url)
    })
  // Rest of code
}
```

Reply:
Yup. It should just work :)